pep-0282.txt

```diff
@@ -170,13 +174,13 @@ This is done through a module-level function::
 Levels
 ======
 
-The logging levels, in increasing order of importance, are::
+The logging levels, in increasing order of importance, are:
 
-    DEBUG
-    INFO
-    WARN
-    ERROR
-    CRITICAL
+* DEBUG
+* INFO
+* WARN
```

Comment:
After the PEP was written, `WARN` was replaced by `WARNING` as preferable.though `WARN` remains as an alias. Perhaps this should be reflected here too?

Reply:
We can defer to your judgement as PEP author, but [per PEP 1](https://peps.python.org/pep-0001/#pep-maintenance),

> In general, PEPs are no longer substantially modified after they have reached the Accepted, Final, Rejected or Superseded state. Once resolution is reached, a PEP is considered a historical document rather than a living specification. Formal documentation of the expected behavior should be maintained elsewhere, such as the [Language Reference](https://docs.python.org/3/reference/index.html) for core features, the [Library Reference](https://docs.python.org/3/library/index.html) for standard library modules or the [PyPA Specifications](https://packaging.python.org/en/latest/specifications/) for packaging.

Presumably there are other things that are out of date here too, so IMO there isn't a need to update this just because it happened to be touched in a syntax update to ensure the PEP renders as intended.

I just merged #2702 which adds a new banner to point readers to where they can find the up to date, canonical documentation for the feature proposed by the PEP once its Final, which we could use to add a prominent link directing readers to the stdlib [logging module reference](https://docs.python.org/3.12/library/logging.html).

Reply:
Agreed.